dbt/include/snowflake/macros/adapters.sql

```diff
@@ -147,7 +149,7 @@
     {# -- Gated for performance reason. If you don't want Iceberg, you shouldn't pay the
        -- latency penalty. #}
     {% if adapter.behavior.enable_iceberg_materializations.no_warn %}
-      select all_objects.*, is_iceberg as "is_iceberg"
+      select all_objects.*, is_iceberg
```

Comment:
I tested in by manually place `QUOTED_IDENTIFIERS_IGNORE_CASE = true` and `QUOTED_IDENTIFIERS_IGNORE_CASE = false` and then verifying it didn't matter. This is a more predictable way to handle metadata query column names.